frontend/src/pages/destructionlist/review/DestructionListReview.tsx

```diff
@@ -183,7 +186,10 @@ export function DestructionListReviewPage() {
   /**
    * Returns the button to show in the secondary navigation (top bar).
    */
-  function getSubmitDestructionListButton(): ButtonProps {
+  function getSubmitDestructionListButton(): ButtonProps | null {
+    if (!user?.role.canReviewDestruction && !user?.role.canReviewFinalList) {
+      return null; // TODO: Implement mark ready
+    }
```

Comment:
Maybe we need a test that the co-reviewer doesn't have the button

Reply:
- [ ] add test